modules/features/discover/src/main/res/layout/collection_header.xml

```diff
@@ -0,0 +1,63 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="match_parent"
+    android:layout_marginVertical="8dp"
+    android:layout_marginLeft="8dp"
+    android:clickable="true"
+    android:focusable="true">
+
+    <com.google.android.material.card.MaterialCardView
+        android:layout_width="179dp"
+        android:layout_height="match_parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
```

Comment:
Could these be combined? I could be wrong but I don't think it needs a root view.

```suggestion
<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="179dp"
    android:layout_height="match_parent"
    android:layout_marginVertical="8dp"
    android:layout_marginLeft="8dp"
    android:clickable="true"
    android:focusable="true">
```